src/Context/MetaTagContext.php

```diff
@@ -44,29 +44,34 @@ public function checkForOgMetaTags()
 
         $ogTitleNode = $this->getSession()->getPage()->find('css', "meta[property='og:title']");
         if (!$ogTitleNode) {
-            throw new \Exception("Can not find property og:title");
+            throw new \Exception("Can not find property og:title on $curUrl");
```

Comment:
Here we can use a more specific SPL exception. `\Exception` is too generic. This applies to all other `\Exception`s in this method/class